src/main/java/org/medicmobile/webapp/mobile/SettingsStore.java

```diff
@@ -209,7 +209,7 @@ public SettingsException(String message) {
 }
 
 class IllegalSettingsException extends SettingsException {
-
+	public static final long serialVersionUID = -6920159047512297868L;
```

Comment:
Maybe worth adding a comment to explain this serial number

`// see https://docs.pmd-code.org/pmd-doc-6.36.0/pmd_rules_java_errorprone.html#missingserialversionuid`

And can you please update the link in L203 to `https://docs.pmd-code.org/pmd-doc-6.36.0/pmd_rules_java_errorprone.html#missingserialversionuid`?

Reply:
Yes good point! I made both links point to the v7.1.0 of the docs since that's what we're using now
https://docs.pmd-code.org/pmd-doc-7.1.0/pmd_rules_java_errorprone.html#missingserialversionuid